packages/client/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './products';
```

Comment:
Can you move this to the end of the file? This messes with the top tsdoc of the module.